packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -178,3 +178,11 @@ export function getInputFieldType(field: GraphQLField<any, any>): GraphQLInputTy
 export function getDeltaQuery(typeName: string) {
   return `sync${pluralize(typeName)}`;
 }
+
+export function getDeltaListType(typeName: string): string {
```

Comment:
-1 for core. Delta will be very specific to datasync

Reply:
I agree, should we put all getDelta* in datasync?

Reply:
I have moved stuff as non destructively as I could, please check